config/schedule.rb

```diff
@@ -20,7 +20,8 @@
 # end
 
 # Learn more: http://github.com/javan/whenever
+job_type :sweeper, "cd :path && RUBYOPT='-W0' bundle exec bin/rails runner -e :environment ':task' :output"
```

Comment:
Decided to go with `RUBYOPT='-W0'` instead of `RUBYOPT='-W:no-deprecated'` in case this repo is deployed to production that may have an older Ruby version.